src/modules/chat_highlight_blacklist_keywords/index.js

```diff
@@ -1,12 +1,14 @@
-const $ = require('jquery');
-const watcher = require('../../watcher');
-const settings = require('../../settings');
-const storage = require('../../storage');
-const html = require('../../utils/html');
-const twitch = require('../../utils/twitch');
-const cdn = require('../../utils/cdn');
-const moment = require('moment');
-const {escape: escapeRegExp} = require('../../utils/regex');
+import $ from 'jquery';
+import watcher from '../../watcher.js';
+import settings from '../../settings.js';
+import storage from '../../storage.js';
+import html from '../../utils/html.js';
+import twitch from '../../utils/twitch.js';
+import cdn from '../../utils/cdn.js';
+import moment from 'moment';
+import regex from '../../utils/regex.js';
+
+const escapeRegExp = regex.escape;
```

Comment:
instead of this, let's change the exports in the regex util to expose the function itself (rather than in default)

Reply:
utils/regix.js exports 2 functions, is the other-one unused?

Reply:
looks like it

Reply:
done, also removed unused code